odl/contrib/fom/supervised.py

```diff
@@ -628,6 +629,46 @@ def psnr(data, ground_truth, normalize=False):
     else:
         return 20 * np.log10(max_true) - 10 * np.log10(mse_result)
 
+
+def noise_power_spectrum(data, ground_truth, radial=False):
+    """Return the Noise Power Spectrum (NPS).
+
+    The NPS is given by the square of the fourier transform of the noise.
+
+    Parameters
+    ----------
+    data : `DiscreteLp`-element
+        Input data or reconstruction.
+    ground_truth : `DiscreteLp`-element
+        Reference to compare ``data`` to.
+    radial : bool
+        If true, compute the radial NPS.
+
+    Returns
+    -------
+    noise_power_spectrum : `DiscreteLp`-element
+        Noise power spectrum. The space is the fourier transform of
```

Comment:
Just repeating the name?
Fourier
and "Fourier transform of the space" isn't really a thing, rather "Fourier space corresponding to ..." or "frequency space" or so.